aarch64 unsupported?

Hello - I'm following [the build guide](https://viperproject.github.io/prusti-dev/dev-guide/development/build.html) with some help from others in the SPL. However, things are not quite working - probably because I am developing on an M1 Mac (`aarch64-apple-darwin` toolchains). `rustup show` tells me my `active toolchain` is `nightly-2021-09-01-aarch64-apple-darwin (overridden by '$HOME/code/prusti-dev/rust-toolchain')`. My `RUST_SYSROOT` env var is set to `$HOME/.rustup/toolchains/nightly-2021-09-01-aarch64-apple-darwin`. Here's what I get from the build script:

```
warning: test-crates v0.1.0 ($HOME/code/prusti-dev/test-crates) ignoring invalid dependency `prusti` which is missing a lib target
   Compiling libc v0.2.101
   Compiling autocfg v1.0.1
   Compiling cfg-if v1.0.0
   Compiling proc-macro2 v1.0.29
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.75
   Compiling log v0.4.14
   Compiling memchr v2.4.1
error[E0463]: can't find crate for `std`

For more information about this error, try `rustc --explain E0463`.
error[E0463]: can't find crate for `core`

error: could not compile `libc` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

`can't find crate for std` sounds seriously wrong. Anyway, if anyone has any ideas, let me know. Otherwise I'll try on an `x86_64` machine.

--- a/rust-toolchain
+++ b/rust-toolchain
@@ -1,4 +1,4 @@
 [toolchain]
 channel = "nightly-2021-09-01"
-components = [ "rustc-dev", "llvm-tools-preview" ]
+components = [ "rustc-dev", "llvm-tools-preview", "rust-std" ]
 profile = "minimal"

